modules/core/shared/src/main/scala/lucuma/core/model/CatalogInfo.scala

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats._
+import eu.timepit.refined._
+import eu.timepit.refined.cats._
+import eu.timepit.refined.collection.NonEmpty
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.core.enum.CatalogName
+import monocle.Focus
+import monocle.Lens
+
+final case class CatalogInfo(catalog: CatalogName, id: NonEmptyString, objectType: String)
```

Comment:
I would have expected `objectType` to be, maybe, an `Option[NonEmptyString]`.  Do we always get an `objectType` from all the catalogs?